### DISCUSSION TITLE
Send keys from script

### DISCUSSION THREAD

**Discussion by Vince-1024** [EDCD/EDDI#2521] [EDDI Standalone Q&A] [answered]
HI,

Is it possible to trigger a 'send key' or to use an Elite bind key command from a script ?

If not, is it possible to execute a VBS or PowerShell from script like the 'Play' command for sounds ?

Thanks

**Comment by Tkael** [comment]
Hi there! EDDI does not currently have this capability. That said, if you use EDDI as a plugin for VoiceAttack then you can trigger actions in response to any supported EDDI event.

**Comment by Vince-1024** [comment]
Thanks

I will do this from VoiceAttack.

Do you plan to implement it in further release ?

Vincent.

**Reply by Tkael** [reply, marked as answer]
I don't expect to. VoiceAttack works very well and I have many other areas where I can still focus my energy.
